data/SIMS Activation Log.csv

```diff
@@ -0,0 +1,34 @@
+Year,Country,Code,Response,Support,Significance (1-5; 5 as most significant activations),Start Date,End Date
```

Comment:
Hi,

The spreadsheet is located in the SIMS Google Drive folder name: 'SIMS Activation Log'. It was originally started by Kat and has now passed on to me to maintain. I can ping you if you dont have the direct link. 

I decided not to do an AJAX call to the Google Sheets just yet as it contains names of people deployed etc. I agree this is the way to go instead of manually pulling through an edited csv each time. @simonbjohnson mentioned something about an intermediary step being thought about for this very reason. 

The severities were originally devised by our team sometime last year to explain SIMS commitment to other teams. They could be seen to be confusing to the general viewer and maybe should be  removed as do not add direct value. The levels were decided as (for the entire response):
5 - A full time person coordinating, deployed and remote for all/majority of the response
1 - Less than 1 month, equates to 1 day a week for 1 month, remote only.